app/models/appointment.rb

```diff
@@ -58,11 +58,21 @@ def days_overdue
     (Date.today - scheduled_date).to_i
   end
 
+  def scheduled?
+    status.to_sym == :scheduled
+  end
+
   def overdue?
-    status.to_sym == :scheduled && scheduled_date <= Date.today
+     scheduled? && scheduled_date <= Date.today
```

Comment:
```suggestion
    scheduled? && scheduled_date <= Date.today
```

Reply:
Should this be a strict inequality?

Reply:
I'm resolving this since it's unrelated to this PR.